springUsers/src/main/java/com/example/springUsers/Model/User.java

```diff
@@ -0,0 +1,95 @@
+package com.example.springUsers.Model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class User {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.AUTO)
+	private Long id; 
+	
+	private String userId; 
+	
+	@Column(nullable = false)
+	private String first_name;
+	private String last_name; 
+	
+	@Column(nullable = false, unique = true)
+	private String email;
+	
+	private String encryptedPassword; 
+	private boolean emailVerification; 
+	
+	public User() {
```

Comment:
call this something other than User...once you start using Spring Security you'll see that User comes from Security and means something completely different. Better to get ahead of that curve. I'd go with UserEntity